# map to be merged in
nav_msgs/OccupancyGrid map
---
bool success